lib/main_screen.dart

```diff
@@ -0,0 +1,29 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:top_git_rep/design/app_localization.dart';
+import 'package:top_git_rep/screens/search_screen/bloc/github_search_bloc.dart';
+import 'package:top_git_rep/screens/search_screen/bloc/github_search_event.dart';
+import 'package:top_git_rep/services/git_api_provider.dart';
+
+import 'screens/search_screen/search_screen.dart';
+
+class MainScreen extends StatelessWidget {
+  final GitProvider gitProvider = GitProvider();
```

Comment:
would be nice, if you would add service locator to the project

get_it would be a nice option